python/app/g2panel

```diff
@@ -140,6 +145,17 @@ class MainWindow(QtWidgets.QMainWindow):
         #help
         self.actionAbout.triggered.connect(self.show_about)
 
+    def chipInitialize(self):
```

Comment:
I suggest renaming the function to something more descriptive. Maybe something in line with one of these?

```python
check_powerchip()
exit_if_powerchip_is_off()
check_on()
```

Initialize sounds a bit more like we are initializing the chip

Reply:
I have made the change from `chipInitialize()` to `check_powerchip()` for the function name